shared/src/main/scala/mouse/any.scala

```diff
@@ -1,10 +1,10 @@
 package mouse
 
 trait AnySyntax {
-  implicit final def optionSyntaxMouse[A](oa: A): AnyOps[A] = new AnyOps(oa)
+  implicit final def anySyntaxMouse[A](oa: A): AnyOps[A] = new AnyOps(oa)
 }
 
 final class AnyOps[A](val oa: A) extends AnyVal {
-  def |>[B] (f: A => B) = f(oa)
-  def thrush[B] (f: A => B) = f(oa)
+  @inline final def |>[B] (f: A => B) = f(oa)
+  @inline final def thrush[B] (f: A => B) = f(oa)
```

Comment:
I'll accept the PR as is, but I resolved to phase out this pattern of `final` duplicated at both method- and class- level because I think its voodoo. As we touch methods it would be nice to un-final them. 
See https://github.com/typelevel/mouse/pull/41#issuecomment-355866730

Reply:
I'm happy to submit a separate PR to remove them all at once.

Reply:
👍